frontend/device/remarkable/event_map.lua

```diff
@@ -3,5 +3,6 @@ return {
     [105] = "LPgBack",
     [106] = "RPgFwd",
     [116] = "Power",
+    [143] = "Resume",
 }
--- TODO libremarkable has 143 as "wakeup" - don't know what that corresponds to?
+-- 116 is issued when the device gets to sleep, 143 when it's waked up.
```

Comment:
`when it's woken up`, but let's just shorten it to `when it wakes up`?

Reply:
Will do, also just don't merge it yet, I have discovered there's a wakelock that should be set.